cardano-crypto-tests/src/Test/Crypto/VRF.hs

```diff
@@ -44,6 +48,15 @@ tests =
       [ testProperty "bytesToNatural" prop_bytesToNatural
       , testProperty "naturalToBytes" prop_naturalToBytes
       ]
+    , testGroup "ConvertingTypes"
+      [ testProperty "pubKeyToBatchCompat" prop_pubKeyToBatchComopat
+      , testProperty "signKeyToBatchCompat" prop_signKeyToBatchCompat
+      , testProperty "outputToBatchCompat" prop_outputToBatchComat
+      , testProperty "compatibleVerKeyConversion" prop_verKeyValidConversion
+      , testProperty "compatibleOutputConversion" prop_outputValidConversion
+      , testProperty "compatibleSignKeyConversion" prop_signKeyValidConversion
+      , testProperty "compatibleFullConversion" prop_fullValidConversion
```

Comment:
Suggest additional tests:

- Generate keys for both Praos and BatchCompat from the same seed; sign a message with Praos, verify with BatchCompat.
- Same, but the other way around.

We've already tested that all the keys and outputs are the same, but I think it would be good to make this compatibility explicit too.

Reply:
The problem is that the signatures themselves are not compatible across types. Its only the keys (and outputs).

Reply:
Oh, I see. In that case you're right, that test wouldn't make sense.